<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DynamixelServoSetupV1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/png/resources/icon.png</normaloff>:/png/resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>541</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Port:</string>
    </property>
    <widget class="QPushButton" name="pushButton_Connect">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Baud">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_About">
     <property name="geometry">
      <rect>
       <x>469</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>About</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Action">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Action:</string>
    </property>
    <widget class="QPushButton" name="pushButton_Scan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_MaxServos">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>253</number>
     </property>
     <property name="value">
      <number>18</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ChangeId">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Change ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SetPos">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Set position</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ServoNr">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Srv nr:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ReadParam">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Read param</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_ServoNr">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>253</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_GenLog">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>541</width>
      <height>711</height>
     </rect>
    </property>
    <property name="title">
     <string>Log:</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>521</width>
       <height>631</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Mono</family>
      </font>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="cursorWidth">
      <number>5</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Clear">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>88</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
